cmd/bnsapi/main.go

```diff
@@ -83,6 +83,7 @@ func run(conf Configuration) error {
 	rt.Handle("/account/domains", &handlers.AccountDomainsHandler{Bns: bnscli})
 	rt.Handle("/account/accounts", &handlers.AccountAccountsHandler{Bns: bnscli})
 	rt.Handle("/account/accounts/", &handlers.AccountAccountsDetailHandler{Bns: bnscli})
+	rt.Handle("/cash/balances", &handlers.CashBalanceHandler{Bns: bnscli})
```

Comment:
Please add `/cash/balances` to the docs on the endpoint `/`.

Reply:
I will make this code more functional in the future. It is very easy to forget to add this one.